pkg/config/config.go

```diff
@@ -259,9 +280,37 @@ func (lci *LocalConfigInfo) SetConfiguration(parameter string, value interface{}
 
 }
 
-// DeleteConfigDir Deletes the config directory with the config file
-func (lci *LocalConfigInfo) DeleteConfigDir() error {
-	return os.RemoveAll(filepath.Dir(lci.Filename))
+// DeleteConfigDirIfEmpty Deletes the config directory if its empty
+func (lci *LocalConfigInfo) DeleteConfigDirIfEmpty() error {
+	configDir := filepath.Dir(lci.Filename)
+	_, err := lci.fs.Stat(configDir)
+	if os.IsNotExist(err) {
+		// if the config dir doesn't exist then we dont mind
+		return nil
+	} else if err != nil {
+		// possible to not have permission to the dir
+		return err
+	}
+
+	f, err := lci.fs.Open(configDir)
+	if err != nil {
+		return err
+	}
+	defer f.Close()
+
+	_, err = f.Readdir(1)
```

Comment:
why (1) ?

Reply:
Reads only at most 1 files from the dir to check id there is any